src/test/java/hudson/plugins/jobConfigHistory/JobConfigHistoryRootActionTest.java

```diff
@@ -122,7 +122,7 @@ public void testGetJobConfigs() throws Exception {
 
         //one project and one deleted project.
         jenkinsRule.createFreeStyleProject("Test2").delete();
-        assertEquals(6, createSut().getJobConfigs("").size());
+        assertEquals(5, createSut().getJobConfigs("").size());
```

Comment:
No idea why this change of behavior. I don't know enough this plugin to understand what is happening

Reply:
Thanks for the PR, I will take a look and try to find out why this change happened

Reply:
Took me a while, but I figured out that with https://github.com/jenkinsci/jenkins/pull/8528 there is one less config created. So the deletion will create one config change less. And since this PR was introduced in Jenkins 2.426 this changes now also the test. So actually this is a wanted change. Thanks for the PR!